### ISSUE TITLE
OpenGlasses compatibility

### ISSUE THREAD

**Issue by kappa-maintainer** [Laike-Endaril/Dynamic-Stealth#91]
When install Dynamic Stealth and OpenGlasses, mob's health bar is rendered as red in thermal view and become unreadable.
![2020-01-28_10 21 12](https://user-images.githubusercontent.com/11038263/73230497-83d84c80-41b8-11ea-8592-bca56477a240.png)

**Comment by Laike-Endaril** [comment]
I have a feeling they might be using a shader, in which case this will have no effect...but if you have a chance, maybe you can test this build for me and see if it fixes the issue:

[DynamicStealth-1.12.2.101a.zip](https://github.com/Laike-Endaril/Dynamic-Stealth/files/4124634/DynamicStealth-1.12.2.101a.zip)

**Comment by kappa-maintainer** [comment]
Not working. Entity fading feature still works in thermal view, so I guess OpenGlasses is using a shader or something. Maybe I should simply disable targeting hud and get mob's action by observing them.

**Comment by Laike-Endaril** [comment]
The targeting HUD is the part which is working correctly (the arrows and text overlay)

It's the OPHUD (or On-Point HUD) which is displaying in red

But yes, I haven't thought of a good solution I can quickly implement, so if you want an immediate fix you'll likely have to disable the on-point HUD.  I'll leave this open and hopefully come up with a better solution at some point

**Comment by Laike-Endaril** [comment]
The first time I looked into this, I didn't realize there were 2 different OpenGlasses mods...

In any case, I've managed native compatibility.

Just note that the glasses block the glow effect, which will ofc mean no targeting glow or glow from soul sight.  This is a "feature" of the thermal vision and I can't reasonably do anything about that part from my end (same thing happens for vanilla glow from any source, eg. spectral arrows).
